src/components/core/header.js

```diff
@@ -90,20 +91,23 @@ const activeLinkStyle = color => ({
 const MenuElement = ({ label, href, slug, color, ...props }) => (
   <MenuElementWrapper {...props}>
     {href ? (
-      <a href={href} css={linkStyle} target="_blank" rel="noopener noreferrer">
+      <a href={href} css={linkStyle} onClick={() => logAmplitudeEvent('Link to Developer Console clicked', {})} target="_blank" rel="noopener noreferrer">
```

Comment:
The `<MenuElementWrapper/>` component aimed to be generic. I'd suggest rephrasing the event name from  `Link to Developer Console clicked` to `External link clicked` with `href` passed as an additional event param.